src/umpire/op/OpenMPTargetMemsetOperation.hpp

```diff
@@ -21,9 +21,6 @@ class OpenMPTargetMemsetOperation : public MemoryOperation {
    */
   void apply(void* src_ptr, umpire::util::AllocationRecord* src_allocation,
              int value, std::size_t length);
-
- private:
-  int m_device_id;
 };
```

Comment:
Note to @davidbeckingsale: The compiler warned about this when I was building with openMP enabled in order to create a new replay output test file.  So, I removed it since it is not being used.